docs/developer/advanced/upgrading-nodejs.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[upgrading-nodejs]]
+== Upgrading Node.js
+
+{kib} requires a specific Node.js version to run.
+When running {kib} from source, you must have this version installed locally.
+
+The required version of Node.js is listed in several different files throughout the {kib} source code.
+Theses files must be updated when upgrading Node.js:
+
+  - {kib-repo}blob/{branch}/.ci/Dockerfile[`.ci/Dockerfile`] - The version is specified in the `NODE_VERSION` constant.
+    This is used to pull the relevant image from https://hub.docker.com/_/node[Docker Hub].
+    Note that Docker Hub can take 24+ hours to be updated with the new images after a new release of Node.js, so if you're upgrading Node.js in Kibana _just_ after the official Node.js release, you have to check if the new images are present on Docker Hub.
+    If they are not, and the update is urgent, you can skip this file and update it later once Docker Hub has been updated.
+  - {kib-repo}blob/{branch}/.node-version[`.node-version`]
+  - {kib-repo}blob/{branch}/.nvmrc[`.nvmrc`]
+  - {kib-repo}blob/{branch}/package.json[`package.json`] - The version is specified in the `engines.node` field.
+
+See PR {kib-repo}pull/86593[#86593] for an example of how the Node.js version has been upgraded previously.
+
+In the 6.8 branch, the `.ci/Dockerfile` file does not exist, so when upgrading Node.js in that branch, just skip that file.
+
+=== Backporting
+
+The following rules are not set in stone.
+Use best judgement when backporting.
+
+Currently version 7.11 and newer run Node.js 14, while 7.10 and older run Node.js 10.
+Hence, upgrades to either Node.js 14 or Node.js 10 shold be done as separate PRs.
+
+==== Node.js patch upgrades
+
+Typically, you want to backport Node.js *patch* upgrades to all supported release branches that run the same _major_ Node.js version:
```

Comment:
major should be in bold instead of italics

Reply:
Addressed in #87976